src/clients/FlexbaseClient.Patron.ts

```diff
@@ -16,4 +31,68 @@ export class FlexbaseClientPatron extends FlexbaseClientBase {
             return [];
         }
     }
+
+    async addPatron(patronData: IPatronData): Promise<IResponseUpdatePatron> {
+        try {
+          const response = await this.client
+            .url('/clients')
+            .post(patronData)
+            .json();
+    
+          return {
+            patron: {
+              id: response.id,
+              name: response.name,
+              companyId: response.companyId,
+              tenantId: response.tenantId,
+              address: {
+                street1: response.address,
+                city: response.city,
+                country: response.country,
+                state: response.state,
+                postalCode: response.postalCode,
+              }
+            },
+            success: true,
+            message: 'You have a new patron!',
+          };
```

Comment:
don't return `success` or `message` or `error` just the patron. So just transpose the response into a `Patron` and return that.

Reply:
I got it